src/main/java/org/spongepowered/common/mixin/core/server/MixinNetHandlerPlayServer.java

```diff
@@ -187,4 +193,32 @@ public void processCommandBlock(C17PacketCustomPayload packetIn, CallbackInfo ci
 
     }
 
+    @Inject(method = "processPlayer", at = @At(value = "FIELD", target = "net/minecraft/network/NetHandlerPlayServer.hasMoved", ordinal = 2), cancellable = true, remap = false)
```

Comment:
No, read the javadoc on the `@Inject` annotation. remap is whether it should remap the method/class/field names in production (an obfuscated environment). It needs to convert the friendly names to SRG names.